src/providers/persistence/persistence.ts

```diff
@@ -883,6 +883,14 @@ export class PersistenceProvider {
   getCardFastTrackEnabled() {
     return this.storage.get(Keys.CARD_FAST_TRACK_ENABLED);
   }
+
+  setOnboardingFlowFlag(value: string) {
```

Comment:
Why it's needed another Onboarding flag?

Reply:
The other flag is to show the keyOnboardingSlides that are different from the "feature education" of the new onboarding. Those slides can appear in case the user deletes all their keys and has already completed onboarding.